### PR TITLE
Updated Chinese Translate.

### PR DESCRIPTION
**Pull request by BigeYoung**

### CODE REVIEW

File: content/zh/docs/community/localization.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+翻译后的内容需要位于 `content/[LANG]/docs`”目录中。它的网址应与英语来源相同。例如，翻译
+韩语简介，复制对应的英语简介可能会有帮助，你可以这样做：
+
+```sh
+mkdir -p content/ko/docs/intro
+cp content/en/docs/intro/install.md content/ko/docs/intro/install.md
+```
+
+然后可以将新文件中的内容翻译成其他语言。
+
```
Comment (bridgetkromhout) [inline content/zh/docs/community/localization.md:70]: Can you please add the change from https://github.com/helm/helm-www/pull/788/files? Right here, we need to add this information:

> Do not add an untranslated copy of an English file to `content/[LANG]/`.
Once a language exists on the site, any untranslated pages will redirect to
English automatically. Translation takes time, and you always want to be
translating the most current version of the docs, not an outdated fork.

> Make sure you remove any `aliases` lines from the header section. A line like
`aliases: ["/docs/using_helm/"]` does not belong in the translations. Those
are redirections for old links which don't exist for new pages.

File: content/zh/docs/helm/helm_install.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
+---
```
Comment (bridgetkromhout) [inline content/zh/docs/helm/helm_install.md:1]: This is an auto-generated file. We need to figure out the best way to handle this; for now, can you put it in a different pull request, so we can merge the normal docs files without having to wait?

**Comment by bridgetkromhout** [comment]
Hi, @BigeYoung! Thank you very much for translating these docs. If you can fix those couple of problems I mentioned, we can merge this pull request! Thanks.

**Comment by bridgetkromhout** [comment]
Hi, @BigeYoung - while we were waiting to hear back from you, we had another PR come in which means we now are additionally hoping for you to correct the merge conflict. Thank you!

**Comment by BigeYoung** [comment]
I have completed the correction as you requested, please have a look.
我已按您的要求完成更正，请过目。

File: content/zh/docs/community/localization.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
+---
+title: "本地化 Helm 文档"
+description: "本指南说明了如何本地化 Helm 文档。"
+aliases: ["/docs/localization/"]
```
Comment (bridgetkromhout) [inline content/zh/docs/community/localization.md:4]: Please remove this alias; it will not do what you think it will do. (It will redirect the English-language localization page to this new page, which is not what we want.)

**Comment by bridgetkromhout** [comment]
Hi, @BigeYoung! Thank you again. There is just one line to remove (aliases: ["/docs/localization/"]) and then we can merge this pull request! Please make that change.

**Comment by bridgetkromhout** [comment]
Hi, @BigeYoung! I am sorry to tell you it looks like you accidentally added commits without the DCO signoff - can you sign those? Thanks.

**Comment by bridgetkromhout** [comment]
Hi, @BigeYoung - we want to merge your pull request. Can you please follow these directions to fix your sign-off on these two commits? https://github.com/helm/helm-www/pull/813/checks?check_run_id=1131783416 

Thank you!

**Comment by bridgetkromhout** [comment]
I will close this PR at this time, due to the merge conflicts and the missing DCO signoff. Please do bring your fork up to date and submit a new PR if you would like to. Thanks!
